scripts_python/1_data_creation/dwi_ml_tractoflow_extrasteps.py

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env python
```

Comment:
Not sure this should be included. If anyone wants to add post-processing steps to tractoflow they'll add them themselves.

Reply:
Ok. Are there not any post-processing steps that could be useful to everyone doing ML? The goal is to avoid again the situation where everyone has a copy of the same parts of code on their computer.

@ppoulin91 , you know more your project vs Antoine. Was there similar post-processing being done?